presto-spi/src/main/java/com/facebook/presto/spi/ConnectorTableLayout.java

```diff
@@ -51,7 +55,8 @@ public ConnectorTableLayout(
             Optional<ConnectorNodePartitioning> nodePartitioning,
             Optional<Set<ColumnHandle>> streamPartitioningColumns,
             Optional<DiscretePredicates> discretePredicates,
-            List<LocalProperty<ColumnHandle>> localProperties)
+            List<LocalProperty<ColumnHandle>> localProperties,
+            TableStatistics statistics)
```

Comment:
Why not optional argument?
E.g. separate c-tor with default value for `statistics` = `EMPTY_STATISTICS`? It would make transition easier.
Even for future it should not be mandatory to provide statistics, so we should not bother connectors like `atop` knowing about `EMPTY_STATISTICS` existence.